backend/controllers/Auth.js

```diff
@@ -0,0 +1,32 @@
+const router = require("express").Router();
+const User = require("../models/User");
+
+const fields = {
+  firstName: "first name",
+  lastName: "last name",
+  email: "email",
+};
+router.post("/register", async (req, res) => {
+  try {
+    const { firstName, lastName, email } = req.body;
+    const newUser = new User({
+      firstName,
+      lastName,
+      email,
+    });
```

Comment:
Could we simplify this further? Maybe:

```suggestion
    const newUser = new User(req.body);
```